src/server/core/journey/route/finder.ts

```
@@ -58,214 +73,251 @@ export async function findRoutes(
+            if (edges.length > 0) {
+              const isSignificantlyDifferent = !paths.some((existingPath) =>
+                areSimilarPaths(path, existingPath.path, 0.7)
+              );
+
+              if (isSignificantlyDifferent) {
+                paths.push({ path, edges });
+              }
+            }
+          }
+        }
+      } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)